c/src/ml-api-inference-single.c

```diff
@@ -1821,6 +1849,40 @@ ml_single_get_property (ml_single_h single, const char *name, char **value)
     /* boolean */
     g_object_get (G_OBJECT (single_h->filter), name, &bool_value, NULL);
     *value = (bool_value) ? g_strdup ("true") : g_strdup ("false");
+  } else if (g_str_equal (name, "input") || g_str_equal (name, "output")) {
+    gchar *dim_str = NULL;
```

Comment:
How about creating a function?

Reply:
Suggestion: TBU, adding 'rank' in GstTensorInfo, so that we can provide exact dim-string from nnstreamer side, not ML-API inside.

Reply:
okay, thank you.